docker-core-network.yaml

```diff
@@ -10,9 +10,18 @@ services:
     volumes:
       - './certs/arbiterToken-databox-network:/run/secrets/DATABOX_NETWORK_KEY'
       - './certs/databox-network.pem:/run/secrets/DATABOX_NETWORK.pem'
+      - '${BCAST_FIFO}:/tmp/relay'
+    command: ["-f", "/tmp/relay"]
     cap_add:
       - NET_ADMIN
 
+  databox-broadcast-relay:
+    image: ${DOCKER_REPO}core-network-relay:${DATABOX_CORE_IMAGE_VERSION}
+    network_mode: "host"
+    volumes:
+      - '${BCAST_FIFO}:/tmp/relay'
+    command: ["-f", "/tmp/relay", "-h", "${BCAST_IP}"]
```

Comment:
tested by setting `BCAST_IP` here to an nonexistent IP

on mac, `BCAST_IP` is the host's IP on the LAN, but from within the container interfaces are all virtualized, such an IP could not be found, which would crash the service previously